requirements-dev.txt

```diff
@@ -4,6 +4,7 @@ pylint==2.4.4
 pytest
 pytest-cov
 python-dotenv
+python-ldap
```

Comment:
Remove.

Reply:
I added this here because Travis CI failed when I did the last code changes. Anyway, I will remove it and I will see if Travis CI still fails of if this is no longer needed